test/Sentry.Diagnostics.DiagnosticSource.Tests/Sentry.Diagnostics.DiagnosticSource.Tests.csproj

```
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' or $(TargetFramework.StartsWith('netstandard2')) or $(TargetFramework.StartsWith('net4')) or $(TargetFramework.StartsWith('net4'))">
+    <ProjectReference Include="../../src/Sentry.Diagnostics.DiagnosticSource/Sentry.Diagnostics.DiagnosticSource.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' OR '$(TargetFramework)' == 'net461'">
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.17" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.17" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="3.1.17" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' OR '$(TargetFramework)' == 'net5.0'">
```

Just revert the logic from the above:

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp2.1' AND '$(TargetFramework)' != 'net461'">
```

Think about next time we add a target here like `net6`. We want one of those sides to be picked up automatically